tools/ci_build/build.py

```diff
@@ -597,7 +603,8 @@ def generate_build_tree(cmake_path, source_dir, build_dir, cuda_home, cudnn_home
             tensorrt_home if args.use_tensorrt else ""),
         # set vars for migraphx
         "-Donnxruntime_USE_MIGRAPHX=" + ("ON" if args.use_migraphx else "OFF"),
-        "-Donnxruntime_MIGRAPHX_HOME=" + (migraphx_home if args.use_migraphx else ""),
+        "-Donnxruntime_MIGRAPHX_HOME=" + \
+        (migraphx_home if args.use_migraphx else ""),
         # By default - we currently support only cross compiling for
```

Comment:
Why these changes? Line length limit is 120 so they look to be unnecessary.

Reply:
These were automatically formatted by VS code, can take these off

Reply:
there's a .flake8 config file in the root directory that sets the line limit to 120. maybe VS code needs to be told about that. our standard is 120 chars for all code.

---
In reply to: [444442986](https://github.com/microsoft/onnxruntime/pull/4287#discussion_r444442986) [](ancestors = 444442986)